### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by lasersox** [google/tracing-framework#619]
I don't see any changes or activity since 2019?

**Comment by stellaraccident** [comment]
I am aware of people still using it, but I don't think the core devs have any further roadmap of places to take it. At least two of us retired our use of it in favor of https://github.com/wolfpld/tracy (which is pretty far superior but not an exact match to some of what this project does).

**Comment by lasersox** [comment]
Thank you @stellaraccident. Tracy looks interesting but it doesn't seem to support web client?

**Comment by stellaraccident** [comment]
Yeah, I'm just one user/developer, but have made the transition to using
the native client. The web client is super convenient, but for my projects,
features trumped.
